### PR TITLE
Added Add, Remove, Index 

### PR DESCRIPTION
**Pull request by Arch-Noize**
Updated the To-Do list project to include the following functions
- Implemented a function for adding a new task (add a new element to the array).
- Implemented a function for deleting a task (remove an element from the array).
- Implemented a function for editing task descriptions.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,57 +1,105 @@
...
+
+/* Find Index */
+
+const findIndex = (e) => {
+  const items = document.querySelectorAll('.item');
+  let index = 0;
+  for (let i = 0; i < items.length; i++) {
+    const text = e.target.parentElement.textContent;
+    if (text === todo[i].desc) {
+      index = i;
```
Comment (rotimiazeez) [inline src/index.js]: Please note that it is required to create a new JavaScript file for the new functionality.
Screenshot from 2023-07-14 18-26-49

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Arch-Noize 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- All hardcoded items from the tasks array are removed. 👍🏽
- A function for adding a new task (add a new element to the array) is implemented. 💯
- A function for editing task descriptions is implemented. 👏🏽
- A function for deleting a task (remove an element from the array) is implemented 👌🏽
- By default, new tasks have the property `completed` set to `false` and the property `index` set to the value of the new array length 🤝
- Deleting a task update all remaining items' indexes, so they represent the current list order and are unique. 😉
- All changes to the To-Do List should be saved in local storage. 😄 
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.js (hunk 1 above)
Comment (Arch-Noize) [reply src/index.js]: Oh I see, thank you for the reply. Do you mean I should separate my functions into a separate "Task" file and export it to index.js?

Comment (Whoistolu) [reply src/index.js]: - Well implemented

**Review by Whoistolu — APPROVED** [review]
### Hi @Arch-Noize 

Great job implementing the recommended changes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**